README.md

```diff
@@ -1,12 +1,12 @@
 # Flask-Phrase
 
-This adapter lets you connect your [Flask](http://flask.pocoo.org/) application to [PhraseApp](https://phraseapp.com) and integrate the powerful [In-Context-Editor](http://demo.phraseapp.com/) into your apps.
+This adapter lets you connect your [Flask](http://flask.pocoo.org/) application to [Phrase](https://phrase.com) and integrate the powerful [In-Context-Editor](http://demo.phrase.com/) into your apps.
```

Comment:
I noticed that `https://demo.phrase.com/` is not working. Asked on slack: https://phraseapp.slack.com/archives/C9A8F7XGD/p1570010923019800